pygments_init.rb

```diff
@@ -0,0 +1,4 @@
+require 'pygments'
+
+# use a custom Pygments installation (directory that contains pygmentize)
+Pygments.start `dirname "$(which pygmentize)"`
```

Comment:
I'm not sure if that actually works as intended. On my system, pygmentize was coming from `python3-pygments` .deb package, thus it points to `/usr/bin/pygmentize`, thus dirname of that will be `/usr/bin`. I assume what is really supposed to be found in this situation is `/usr/lib/python3/dist-packages/pygments`?

Also, in theory this can be written in Ruby without a shell call.

Reply:
> On my system, pygmentize was coming from `python3-pygments` .deb package, thus it points to `/usr/bin/pygmentize`

Yes, I think it _should_ work fine. Executing `which pygmentize` returns `/c/Python27/Scripts/pygmentize` for me (I'm using Git Bash on Windows). And I follow the comment's suggestion (I took this approach from the Asciidoctor docs: [Using a custom Pygments installation](https://asciidoctor.org/docs/user-manual/#using-a-custom-pygments-installation), "directory that contains pygmentize", so I used `dirname`. But I've just discovered that it works even without it, so I'll throw the `dirname` out.

> Also, in theory this can be written in Ruby without a shell call.

I didn't know that, if so, it would be probably better. My knowledge of Ruby is limited 😄  We can replace it with [something like this](https://stackoverflow.com/a/5471032/) if you prefer.

Reply:
Ok, I get the idea, let's leave it as is now. Thanks!